plugins/woocommerce/client/admin/client/marketplace/utils/functions.tsx

```diff
@@ -112,6 +112,11 @@ async function fetchSearchResults(
 	totalPages: number;
 	totalProducts: number;
 } > {
+	// add user locale to search params if not already present
+	if ( LOCALE.userLocale && ! params.get( 'locale' ) ) {
+		params.set( 'locale', LOCALE.userLocale );
+	}
```

Comment:
I see that in `fetchCategories` below, we do the following:

```js
	if ( LOCALE.userLocale ) {
		url.searchParams.set( 'locale', LOCALE.userLocale );
	}
```

But that function does not accept `params`, like this one does, so that seems okay.

In any case, the way we're [using this function](https://github.com/woocommerce/woocommerce/blob/fa1612750920d3392bfa78d360c2129b6262cecd/plugins/woocommerce/client/admin/client/marketplace/components/content/content.tsx), we never set `locale`, so maybe we could remove the `! params.get` check here?

Reply:
> In any case, the way we're [using this function](https://github.com/woocommerce/woocommerce/blob/fa1612750920d3392bfa78d360c2129b6262cecd/plugins/woocommerce/client/admin/client/marketplace/components/content/content.tsx), we never set locale, so maybe we could remove the ! params.get check here?

We don't but wanted to keep it on the safe side. I'm open to changes though if you feel strongly about it

Reply:
No strong preference here

Reply:
Okay, so I'll merge it as is. Maybe this check is not needed now but less chance of overriding things by accident in the future.